src/Modal/modal.style.js

```diff
@@ -1,32 +1,37 @@
 import styled from "styled-components";
+import { Colors } from "../index";
 
-const ModalStyles = styled.div`
-  .container {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background: rgba(0, 0, 0, 0.35);
-    z-index: 10001;
-  }
-
-  .modal-body {
-    max-width: 540px;
-    border-radius: 2px;
-    background-color: #ffffff;
-    box-shadow: 0 0 10px 0 rgba(63, 78, 90, 0.11);
-    transform: translate(-50%, -50%);
-    background: #fff;
-    position: fixed;
-    top: 50%;
-    left: 50%;
+const zIndex = 10;
+export const Overlay = styled.div`
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: ${zIndex};
+  background-color: rgba(0, 0, 0, 0.7);
+`;
 
-    @media (max-width: 800px) {
-      width: 80%;
-      max-width: 331px%;
-    }
-  }
+export const ModalStyles = styled.div`
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: ${zIndex};
 `;
 
-export default ModalStyles;
+export const Content = styled.div`
+  z-index: ${zIndex};
```

Comment:
Where are you getting these stylings from? They dont match our new concept or the old modal?

Our new concept has no box shadow, bigger border radius etc?